frontend/src/views/cluster/ManagedByTemplatesWarning.vue

```diff
@@ -1,45 +1,39 @@
-<!--
-Copyright (c) 2025 Sidero Labs, Inc.
-
-Use of this software is governed by the Business Source License
-included in the LICENSE file.
--->
 <script setup lang="ts">
-import type { Ref } from 'vue'
+import { toRef } from 'vue'
 import { computed, ref } from 'vue'
 import { useRoute } from 'vue-router'
 
 import { Runtime } from '@/api/common/omni.pb'
 import type { Resource } from '@/api/grpc'
-import type { ClusterSpec } from '@/api/omni/specs/omni.pb'
 import { ClusterType, DefaultNamespace, ResourceManagedByClusterTemplates } from '@/api/resources'
 import Watch from '@/api/watch'
 import TAlert from '@/components/TAlert.vue'
 
 export type WarningStyle = 'alert' | 'popup' | 'short'
 
 type Props = {
-  cluster?: Resource<ClusterSpec>
+  resource?: Resource
   warningStyle?: WarningStyle
 }
 
 const props = withDefaults(defineProps<Props>(), {
   warningStyle: 'alert',
 })
 
-const cluster: Ref<Resource<ClusterSpec> | undefined> = ref(props.cluster)
+const propResource = toRef(() => props.resource)
+const routeResource = ref<Resource>()
+const activeResource = computed(() => propResource.value || routeResource.value)
```

Comment:
you don't need to use `toRef` with props in vue 3.5+, they are reactive already

https://vuejs.org/guide/components/props#reactive-props-destructure

the new way of the above stuff is:

```ts
// destructure whatever you want, optionally apply defaults
const { resource, warningStyle = 'alert' } = defineProps<Props>()

const routeResource = ref<Resource>()
const activeResource = computed(() => resource || routeResource.value)
```

you can keep `propResource` name if you want with
```ts
const { resource: propResource, warningStyle = 'alert' } = defineProps<Props>()
```

Reply:
done